### DISCUSSION TITLE
Allow configuring start page

### DISCUSSION THREAD

**Discussion by shanberg** [athensresearch/athens#1021] [Feature Requests]
In some cases, users may not want to start on a new Daily Notes page on startup. For those users, being able to configure what page they start on would improve their experience.

I imagine the options being Daily Notes, [Choose a Page], All Pages, New Blank Page (and eventually New Page from Template), but in the near term the options could just be Daily Notes or New Blank Page, to simplify implementation.

For users unfamiliar with the value of Daily Notes as a notetaking method, we can provide a help notice or tooltip to explain in one or two sentences, and then link out to more detailed documentation.

**Comment by shanberg** [comment]
Another configuration option: "Where I Left Off" or similar

**Reply by Limezy** [reply]
That would be my preferred option
